lib/ansible/module_utils/redfish_utils.py

```diff
@@ -133,11 +133,17 @@ def _find_systems_resource(self, uri):
             if response['ret'] is False:
                 return response
             data = response['data']
-            if data.get(u'Members'):
+            if data.get(u'Members@odata.count') == 1:
                 for member in data[u'Members']:
                     systems_service = member[u'@odata.id']
                     self.systems_uri = systems_service
                     return {'ret': True}
+            if data.get(u'Members@odata.count') > 1:
+                systems_service = []
+                for member in data[u'Members']:
+                    systems_service.append = member[u'@odata.id']
+                self.systems_uri = systems_service
```

Comment:
I am pretty sure the rest of the code will blow up, since it assumes that `systems_uri` is a string and not an array.

Reply:
@tadeboro thank you for the review :) will look in to your comments.